pkg/etcd/etcd.go

```diff
@@ -106,19 +105,19 @@ func (e *ETCD) Test(ctx context.Context, clientAccessInfo *clientaccess.Info) er
 	return fmt.Errorf(msg)
 }
 
-// walDir returns the path to dataDir/member/wal
-func walDir(config *config.Control) string {
-	return filepath.Join(dataDir(config), "member", "wal")
+// etcdDir returns the path to dataDir/db/etcd
+func etcdDir(config *config.Control) string {
```

Comment:
nit: maybe this should be etcdDataDir which will give more information about what this dir is